utils/vlog/vlog_to_model.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+import yosys_exec
+import lxml.etree as ET
+import argparse, re
+import os, tempfile
+from yosys_json import YosysJson
+
+"""
+Convert a Verilog simulation model to a VPR `model.xml`
+"""
+
+parser = argparse.ArgumentParser(description='Convert a Verilog simulation model into a VPR model.xml file')
```

Comment:
I like this trick :-)
```python
parser = argparse.ArgumentParser(description=__doc__.strip())
```

Reply:
Didn't know that :-) Will clean things up